src/DisputeManager.ts

```diff
@@ -212,7 +213,7 @@ function updateRound(
   round.settledPenalties = result.value5
   round.collectedTokens = result.value6
   round.coherentJurors = result.value7
-  round.state = castAdjudicationState(Bytes.fromI32(result.value8))
```

Comment:
So that will result always zero? That why the problem?

Reply:
yupp